app/src/main/java/com/example/sharingang/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.example.sharingang
 
-import androidx.appcompat.app.AppCompatActivity
+import android.content.Intent
 import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.example.sharingang.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        val binding: ActivityMainBinding =
+            DataBindingUtil.setContentView(this, R.layout.activity_main)
+       binding.description = intent.getStringExtra("LAST_DESCRIPTION")
+    }
+
+    fun createNewItem(view: View) {
+        val intent = Intent(this, NewItemActivity::class.java)
+        startActivity(intent)
```

Comment:
And here we should wait for the result of NewItemActivity, the best way seems to be like this: https://stackoverflow.com/a/66345734/8077618